web/src/components/dashboard/SidebarCardsList.test.tsx

```diff
@@ -161,44 +218,66 @@ describe("<SidebarCardsList />", () => {
         municipality: undefined,
         sectorId: undefined,
         existingEmployees: "50",
+        homeBasedBusiness: false,
       }),
     });
 
     it("render cards with a zero count in the For You header when no Fundings and no certifications are present", () => {
       useMockBusiness(mockBusiness);
-      render(
-        <SidebarCardsList
-          topCards={[]}
-          bottomCards={[]}
-          fundings={[]}
-          hiddenFundings={[]}
-          certifications={[]}
-          hiddenCertifications={[]}
-          displayFundings={true}
-          displayCertifications={true}
-        />
-      );
+      renderComponent({
+        fundings: [],
+        certifications: [],
+        displayFundings: true,
+        displayCertifications: true,
+      });
       const forYouCounter = screen.getByTestId("for-you-counter").textContent;
       expect(forYouCounter).toEqual("(0)");
     });
 
     it("render cards with a one count in the For You header when one certification is present", () => {
       useMockBusiness(mockBusiness);
-      render(
-        <SidebarCardsList
-          topCards={[]}
-          bottomCards={[]}
-          fundings={[]}
-          hiddenFundings={[]}
-          certifications={[generateCertification({})]}
-          hiddenCertifications={[]}
-          displayFundings={true}
-          displayCertifications={true}
-        />
-      );
+      const genericCertification = generateCertification({
+        applicableOwnershipTypes: [],
+        isSbe: false,
+      });
+      renderComponent({
+        fundings: [],
+        certifications: [genericCertification],
+        displayFundings: true,
+        displayCertifications: true,
+      });
+      const forYouCounter = screen.getByTestId("for-you-counter").textContent;
+      expect(forYouCounter).toEqual("(1)");
+    });
+
+    it("render cards with a one count in the For You header when one funding is present", () => {
+      useMockBusiness(mockBusiness);
+      const genericFunding = generateFunding({
+        isNonprofitOnly: false,
+        county: [],
+        sector: [],
+        employeesRequired: undefined,
+      });
```

Comment:
This test and the one previous were somewhat flaky. The generator functions were occasionally creating fundings and certifications that would get filtered out before being displayed in the UI. Tried to make a generic funding and certification that would never get filtered.